core/src/main/java/org/bptlab/cepta/utils/notification/NotificationHelper.java

```diff
@@ -5,9 +5,27 @@
 import org.bptlab.cepta.models.internal.delay.DelayOuterClass;
 import org.bptlab.cepta.models.internal.notifications.notification.NotificationOuterClass;
 import org.bptlab.cepta.models.internal.notifications.notification.NotificationOuterClass.Notification;
+import org.bptlab.cepta.models.internal.types.coordinate.CoordinateOuterClass;
 import org.bptlab.cepta.models.internal.types.ids.Ids;
 
 public class NotificationHelper {
+    public static Notification getTrainDelayNotificationFrom(String ceptaID, long delayedSeconds, String delayDetail, long stationId, CoordinateOuterClass.Coordinate coordinate){
```

Comment:
maybe change ceptaID -> ceptaIDString?

Reply:
In its current state the variable is given with the type in front of it so i think thats fine